oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -115,6 +107,7 @@ Include this statement in the README.rst file:
 
     This repository has been archived and is no longer supported—use it at your own risk. This repository may depend on out-of-date libraries with security issues, and security updates will not be provided. Pull requests against this repository will also not be merged.
 
+It is also recommended that you add a brief note about why the repo is being archived, and what it serving as its replacement (where applicable). This may be as simple as a linking to the appropriate DEPR ticket.
```

Comment:
**Author Concern:** Like the other comment, I think this should be required. We require documenting decisions, and the decision to archive seems like a decision worth documenting, even if just in a brief note on the README.